hive-plugin/src/main/java/edu/berkeley/ground/plugins/hive/GroundStoreBase.java

```diff
@@ -96,37 +69,33 @@ public boolean addPartitions(String dbName, String tblName, List<Partition> part
     return false;
   }
 
-  public boolean addPartitions(String dbName,
-                               String tblName,
-                               PartitionSpecProxy partitionSpec,
-                               boolean ifNotExists)
+  public boolean addPartitions(String dbName, String tblName, PartitionSpecProxy partitionSpec, boolean ifNotExists)
```

Comment:
We never enforced 100 char limit in the old formatting. I have googlestyle with 100 char limit I can use that. But changes will be too many